libcloud/test/compute/test_gce.py

```diff
@@ -2900,6 +2911,16 @@ def _regions_us_central1_addresses_lcaddress(self, method, url, body,
                 'regions_us-central1_addresses_lcaddress.json')
         return (httplib.OK, body, self.json_hdr, httplib.responses[httplib.OK])
 
+    def _regions_us_central1_addresses_testaddress(self, method, url, body,
+                                                   headers):
+        if method == 'DELETE':
+            body = self.fixtures.load(
+                'regions_us-central1_addresses_testaddress_delete.json')
```

Comment:
It looks like you had started to test the DELETE path but ended up not to? Either add the file or remove the reference to it.

Reply:
Thanks, I removed the reference to it.